staging/src/k8s.io/apiserver/pkg/storage/cacher/cacher_whitebox_test.go

```diff
@@ -924,7 +937,6 @@ func TestDispatchingBookmarkEventsWithConcurrentStop(t *testing.T) {
 
 		select {
 		case <-done:
-			break
```

Comment:
Why this is removed?

Reply:
because this is an ineffective break,
error from static check:
> ineffective break statement. Did you mean to break out of the outer loop? (SA4011)